src/doc/reference.md

```diff
@@ -2070,6 +2070,9 @@ macro scope.
    trait of the same name. `{Self}` will be replaced with the type that is supposed
    to implement the trait but doesn't. To use this, the `on_unimplemented` feature gate
    must be enabled.
+- `must_use` - on structs and enums, requires that instances are either consumed
+   or passed outside the crate. Using the full version `#[must_use = "message"]`
```

Comment:
two things

1. "consumed or passed outside the crate" is not 100% accurate here. "consumed" means something specific in Rust, I would say something more like "will warn if this value isn't bound to a variable."
2. rather than "full version" I would say "You can additionally pass a message with this syntax <blah> that allows you..."